user_agents/parsers.py

```diff
@@ -165,7 +165,7 @@ def is_tablet(self):
             return True
         if self.os.family.startswith('Windows RT'):
             return True
-        if self.os.family == 'Firefox OS' and 'Mobile' not in self.browser.family:
+        if self.os.family == 'Firefox OS' and 'Tablet;' in self.ua_string:
```

Comment:
Not if we move the changed line 168 to line 164 I think. In any case, could you please add a test case for this? Thanks!